docs/core/tutorials/netcore-hosting.md

```diff
@@ -1,5 +1,5 @@
 ---
-title: Write a custom .NET Core runtime host
+title: Write a custom runtime host
```

Comment:
I think leaving .NET Core in the middle makes more sense here, and just add an empty title suffix.

Reply:
Fixed.